monai/bundle/scripts.py

```diff
@@ -650,6 +650,8 @@ def run(
             Default to "configs/logging.conf", which is commonly used for bundles in MONAI model zoo.
         tracking: enable the experiment tracking feature at runtime with optionally configurable and extensible.
             if "mlflow", will add `MLFlowHandler` to the parsed bundle with default logging settings,
+            if "clearml_stats", will add `ClearMLStatsHandler` to the parsed bundle with default logging settings,
+            if "clearml_image", will add `ClearMLImageHandler` to the parsed bundle with default logging settings,
```

Comment:
Currently, we are testing the `--tracking mlflow` here:
https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_integration_bundle_run.py#L127
If you really want to complete the `--tracking clearml_XXX` feature in this PR, please add some unit tests to your test file according to the above code.
The above bundle test code also tested other logic, you can simplify it with some overrides and run, refer to:
https://github.com/Nic-Ma/MONAI/blob/5821-bundle-interface/tests/test_bundle_infer_workflow.py#L217

Thanks in advance.

Reply:
@Nic-Ma, we want to skip the --tracking feature in this PR then. Please, review the PR at your earliest convenience.

Reply:
Hi @skinan ,

Sure, if you don't want to support the `--tracking` feature in this PR, please restore your changes in file `monai/bundle/utils.py` and `monai/bundle/scripts.py`.
Others look good to me now.

Thanks for your quick update.

Reply:
@Nic-Ma , I have restored my changes. Done ! Please, make a final review.